gammapy/estimators/points/tests/test_lightcurve.py

```diff
@@ -634,3 +634,30 @@ def test_lightcurve_estimator_map_datasets():
     assert_allclose(table["norm_err"][0], [0.031508], rtol=1e-2)
     assert_allclose(table["counts"][0], [[2205, 2220]])
     assert_allclose(table["ts"][0], [2557.346464], rtol=1e-2)
+
+
+@requires_data()
+@requires_dependency("iminuit")
+def test_recompute_ul():
```

Comment:
Maybe the test could also check that it recovers a sensible `ul` with this technique compared to the one initially computed?

Reply:
@AtreyeeS Can you still add this self consistency check?

Reply:
Can you suggest a good check? The value changes from `3.2` to `3.77`  in going from `2` to `4` sigma, which I think is sensible... But what would be a good automatic check?

Reply:
check if `.recompute_ul(n_sigma_ul=2)` is close to 3.2 with this method (it may be different if the default profile scan sampling is not good enough)